sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/TranslationOperationLiveTests.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.DocumentTranslation.Tests
+{
+    public class TranslationOperationLiveTests : DocumentTranslationLiveTestBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TranslationOperationLiveTests"/> class.
+        /// </summary>
+        /// <param name="isAsync">A flag used by the Azure Core Test Framework to differentiate between tests for asynchronous and synchronous methods.</param>
+        public TranslationOperationLiveTests(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        private List<TestDocument> oneDocumentList = new List<TestDocument>
+        {
+            new TestDocument("Document1.txt", "First english test document"),
+        };
+
+        private List<TestDocument> twoDocumentsList = new List<TestDocument>
+        {
+            new TestDocument("Document1.txt", "First english test document"),
+            new TestDocument("File2.txt", "Second english test file"),
+        };
+
+        [RecordedTest]
+        public async Task SingleSourceSingleTargetTest()
+        {
+            Uri source = await CreateSourceContainerAsync(oneDocumentList);
+            Uri target = await CreateTargetContainerAsync();
+
+            var client = GetClient();
+
+            var input = new DocumentTranslationInput(source, target, "fr");
+            var operation = await client.StartTranslationAsync(input);
```

Comment:
seeing this everywhere make me think that we are making a right decision by having a single and a list of `DocumentTranslationInput`